src/se/vidstige/jadb/JadbDevice.java

```diff
@@ -45,13 +47,7 @@ private State convertState(String type) {
 
     private Transport getTransport() throws IOException, JadbException {
         Transport transport = transportFactory.createTransport();
-        if (serial == null) {
-            transport.send("host:transport-any");
-            transport.verifyResponse();
-        } else {
-            transport.send("host:transport:" + serial);
-            transport.verifyResponse();
-        }
+        send(transport, serial == null ? "host:transport-any" : "host:transport:" + serial );
```

Comment:
here a `transport.verifyResponse()` is missing. This should break a test? :thinking:

Reply:
I've replaced calls to Transport.send() and Transport.verifyResponse() with a call to JadbDevice.send() that calls exactly [those two methods](https://github.com/vidstige/jadb/blob/3db1b31a0c4f5e74503cf43bbdcf1d1dc51b8ab5/src/se/vidstige/jadb/JadbDevice.java#L182).